openLuup/scenes.lua

```diff
@@ -1,6 +1,6 @@
 local ABOUT = {
   NAME          = "openLuup.scenes",
-  VERSION       = "2018.04.16",
+  VERSION       = "2018.05.11",
```

Comment:
You've identified the root cause of this issue, but the solution is not complete.  

In particular, the same problem would arise if the global scene prolog routine (not a feature of Vera) rejected the scene run request.  I also need to consider what to do for 'paused' scenes.

The variable 't' can, conceptually, be a timer OR a trigger, and they should not have run times added.

So I will fix this, but not in this manner.